e2e/cypress/integration/multiProject.spec.js

```diff
@@ -148,6 +148,44 @@ describe('Render project cards', () => {
       `${username}/${multiProjectsRepoName}/${multiProjectName}`,
     )
   })
+
+  it('should handle subproject names with URL invalid characters', () => {
+    const username = getFakeUsername()
+    const email = faker.unique(faker.internet.email)
+    const password = '123456'
+
+    cy.createUser(username, email, password)
+    cy.visit('/')
+    cy.get('[data-cy=user-menu]')
+
+    cy.forceVisit('/projects/new')
+
+    const projectWithInvalidCharsURL =
+      'https://github.com/eulerlab/open-visual-stimulator'
```

Comment:
Let's start using https://github.com/kitspace-test-repos for all our test repos.